compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -2203,7 +2203,18 @@ public BLangNode transform(TrapExpressionNode trapExpressionNode) {
     @Override
     public BLangNode transform(ReceiveActionNode receiveActionNode) {
         BLangWorkerReceive workerReceiveExpr = (BLangWorkerReceive) TreeBuilder.createWorkerReceiveNode();
-        workerReceiveExpr.setWorkerName(createIdentifier(receiveActionNode.receiveWorkers().name()));
+        Node receiveWorkers = receiveActionNode.receiveWorkers();
+        Token workerName;
+        if (receiveWorkers.kind() == SyntaxKind.SIMPLE_NAME_REFERENCE) {
+            workerName = ((SimpleNameReferenceNode) receiveWorkers).name();
+        } else {
+            // TODO: implement multiple-receive-action support
+            Location receiveFieldsPos = getPosition(receiveWorkers);
+            dlog.error(receiveFieldsPos, DiagnosticErrorCode.MULTIPLE_RECEIVE_ACTION_NOT_YET_SUPPORTED);
+            workerName = NodeFactory.createMissingToken(SyntaxKind.IDENTIFIER_TOKEN,
+                    NodeFactory.createEmptyMinutiaeList(), NodeFactory.createEmptyMinutiaeList());
```

Comment:
use `createIdentifier(Location pos, Token token)` method to give a name to the missing identifier. otherwise will cause a null pointer exception at the later process.

Reply:
I see. Then we have to do the same here also? 
https://github.com/ballerina-platform/ballerina-lang/blob/67c6af3df6e9df4db76e2d3300111ca3d0199221/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java#L583-L584

Reply:
Ah you are calling `createIdentifier(Token token)` method at the end which will call `createIdentifier(Location pos, Token token)`. So its fine no changes needed